core/domain/exp_domain.py

```diff
@@ -2180,9 +2216,18 @@ def extract_content_id_from_choices(html: str) -> str:
                         # All rule inputs for ItemSelectionInput will be
                         # migrated from SetOfHtmlString to
                         # SetOfTranslatableHtmlContentIds.
+                        # Ruling out the possibility of Any other type
+                        # for mypy type checking because for interaction
```

Comment:
mypy --> MyPy, I commented on this elsewhere. Please fix throughout your PR(s).

Reply:
changed